.circleci/config.yml

```diff
@@ -668,16 +668,9 @@ jobs:
       - run:
           name: Build development contracts
           command: |
-            for contract_dir in ./contracts/*/; do
-              name=$(basename $contract_dir)
-              echo "Building $name ..."
-              docker run --volumes-from with_code cosmwasm/rust-optimizer:0.10.5 ./contracts/$name
-              docker cp with_code:/code/artifacts/$name.wasm ./artifacts/$name.wasm
-            done
-      - run:
-          name: Create checksums
-          working_directory: artifacts
-          command: sha256sum *.wasm > checksums.txt && cat checksums.txt
+            echo "Building all contracts under ./contracts"
+            docker run --volumes-from with_code cosmwasm/rust-optimizer:0.10.6 ./contracts/*/
+            docker cp with_code:/code/artifacts ./artifacts
```

Comment:
Arg, I thought I tested it. But this has a strange behavior:
- When the target folder `./artifacts` does not exist, the folder `artifacts` is copied to `.`
- When the  target folder `./artifacts` does exist, `artifacts` is copied into `./artifacts`

```
$ docker cp with_code:/code/artifacts ./artifacts
$ ls artifacts 
-rw-r--r--   1 me  staff   132K 22 Nov 12:21 burner.wasm
-rw-r--r--   1 me  staff   393B 22 Nov 12:26 checksums.txt
-rw-r--r--   1 me  staff   178K 22 Nov 12:22 hackatom.wasm
-rw-r--r--   1 me  staff   135K 22 Nov 12:23 queue.wasm
-rw-r--r--   1 me  staff   190K 22 Nov 12:25 reflect.wasm
-rw-r--r--   1 me  staff   223K 22 Nov 12:26 staking.wasm
$ docker cp with_code:/code/artifacts ./artifacts
$ ls artifacts                                    
drwxr-xr-x   8 me  staff   256B 22 Nov 12:26 artifacts
-rw-r--r--   1 me  staff   132K 22 Nov 12:21 burner.wasm
-rw-r--r--   1 me  staff   393B 22 Nov 12:26 checksums.txt
-rw-r--r--   1 me  staff   178K 22 Nov 12:22 hackatom.wasm
-rw-r--r--   1 me  staff   135K 22 Nov 12:23 queue.wasm
-rw-r--r--   1 me  staff   190K 22 Nov 12:25 reflect.wasm
-rw-r--r--   1 me  staff   223K 22 Nov 12:26 staking.wasm
```

Using `docker cp with_code:/code/artifacts .` seems to be more stable. Could you update this?

Reply:
Sure. I'll do a new small PR to fix this.

Reply:
Fixed in #633.